DataFormats/Math/src/classes.h

```diff
@@ -3,6 +3,8 @@
 #define SMATRIX_USE_CONSTEXPR
 #include <Math/SVector.h>
 #include <Math/SMatrix.h>
+#include <Math/MinimTransformVariable.h>
+#include <TGraphDelaunay2D.h>
```

Comment:
@slava77 , these can be avoid if we replace the `ROOT::Math::*` pattern.

Reply:
do you mean the patterns used in ```DataFormats/Math/src/classes_def.xml``` ?
There was some cleanup of dictionaries by the core team a while back to drop the wildcards.
Can that be revived/completed?

Reply:
yes the pattern in `DataFormats/Math/src/classes_def.xml` . I am testing the following right now.
```
-  <class pattern="std::vector<ROOT::Math::*>" />
+  <class pattern="std::vector<ROOT::Math::SMatrix<*>>" />
+  <class pattern="std::vector<ROOT::Math::MatRepStd<*>>" />
+  <class pattern="std::vector<ROOT::Math::RowOffsets<*>>" />
+  <class pattern="std::vector<ROOT::Math::SVector<*>>" />
```